### DISCUSSION TITLE
Localized routes

### DISCUSSION THREAD

**Discussion by tomexx** [withastro/roadmap#240] [Proposal]
Hello, is there any project that would lead to localization of routes using the Astro? I believe this is a critical feature that is needed in SEO focused projects.

e.g. these should point to the same page

```
/contact
/de/kontakt
```

Thanks a lot

**Comment by delucis** [comment]
Linking to our internationalization docs for anyone else finding this: https://docs.astro.build/en/guides/internationalization/
